package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "4.0.0-collection-element-removal",
+  "version": "3.1.5-pagination-changes-with-new-arrows",
```

Comment:
I guess this needs to sync with master version, if further changes required version would be 4.x.x-description for pre-release version